### PR TITLE
Updated Docs

### PR DESCRIPTION
**Pull request by lakshyabatman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,79 +1,94 @@
...
 ## Preview Your URL & setup design modes for preview
 
-### Test Your First URL preview
+ 
+
+ ### Installation 
+ -  **Using NPM**
+	 - `npm install @unfurl/elements`
+ - **Using Yarn**
+	 -  `yarn install @unfurl/elements`
```
Comment (mohdaftab93) [inline README.md:37]: For adding a package using yarn,
the command is yarn add packageName
yarn install is used to install all the dependencies of the project.

File: README.md [hunk 2]
```diff
@@ -1,79 +1,94 @@
...
+- **Typescript**
+```typescript
+window.Unfurl.setConfig({  
+    apiToken: 'API_TOKEN'  
+  });
+```
+- **For other environments**
+```javascript
+Unfurl.setConfig({  
+    apiToken: 'API_TOKEN'
```
Comment (mohdaftab93) [inline README.md:52]: change it to 'Your_API_TOKEN'

File: README.md [hunk 3]
```diff
@@ -1,79 +1,94 @@
...
+
+### Getting Started
+Once installation is done, it can be imported and configured as follows.
 
-* Navigate to `apps\demo\src\app\app.component.html` and paste your URL in `<unfurl-preview url="Your URL">` tag.
+- `import '@unfurl/elements'`
+- **Typescript**
+```typescript
+window.Unfurl.setConfig({  
+    apiToken: 'API_TOKEN'
```
Comment (mohdaftab93) [inline README.md:46]: change it to 'Your_API_TOKEN'

File: README.md [hunk 4]
```diff
@@ -1,79 +1,94 @@
...
-        <unfurl-preview url="Your URL" mode="simple"></unfurl-preview>
-        <div class="spacer"></div>
-        <unfurl-preview url="Your URL" mode="compact"></unfurl-preview>
-        <div class="spacer"></div>
-        <unfurl-preview url="Your URL" mode="detailed"></unfurl-preview>
-        <div class="spacer"></div>
-   ```
+```
+
+* Paste your URL in `<unfurl-preview url="Your URL">` tag.
```
Comment (mohdaftab93) [inline README.md:57]: Write here,
use <unfurl-preview></unfurl-preview> inside your template, where you want the link to be previewed. 
Paste your URL in the URL attribute.

**Comment by lakshyabatman** [comment]
Okay sure, on it.

File: README.md (hunk 4 above)
Comment (lakshyabatman) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (lakshyabatman) [reply README.md]: Done.

**Pull request merged**